src/Node.js

```diff
@@ -0,0 +1,48 @@
+  // ### DOM Node Shim
+  //
+  // This represents a superficial form of a DOM node which contains most of
+  // the properties that a regular DOM node would contain, but only the ones
+  // needed by slowparse.
+module.exports = (function(){
+  "use strict";
+
+  function Node(nodeType, nodeName, nodeValue, namespaceURI) {
+    this.nodeType = nodeType || null;
+    this.nodeName = nodeName || null;
+    this.nodeValue = nodeValue || null;
+    this.namespaceURI = namespaceURI || null;
+    this.parentNode = null;
+    this.nextSibling = null;
+    this.attributes = nodeType === Node.ELEMENT_NODE ? [] : null;
```

Comment:
Just make this a straight up `[]`, Slowparse isn't going to try to add attributes to things that can't take attributes in the real DOM.